declarative/src/debug.h

```diff
@@ -23,14 +23,8 @@
 #ifndef DEBUG_H
 #define DEBUG_H
```

Comment:
There's nothing using this file now? I guess this was used earlier and you could enable debug only for qml side.

Reply:
Ah, right, still the includepath = ../src that gets the one from the lib sources. I think it's better not to include the one from the lib sources, so I'm keeping the file, and I've checked that the #include in the declarative sources are including "debug.h" and not "debug_p.h" which the one from the lib.